### PR TITLE
bower: `algoliasearch` v2.x - instead of the bundled vendor script

### PR DESCRIPTION
**Pull request by grimen**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
     "icomoon": "0.0.3",
     "ionicons": "2.0.1",
     "material-design-icons": "2.0.0",
-    "octicons": "3.3.0"
+    "octicons": "3.3.0",
+    "material-design-icons": "2.0.0",
+    "octicons": "3.3.0",
```
Comment (jerone) [inline bower.json]: Something has gone wrong here.

Comment (grimen) [inline bower.json]: Good catch, I will fix it.

Comment (grimen) [inline bower.json]: Corrected.

**Comment by grimen** [comment]
Note: Also in #66

**Pull request merged**